Social Login and Org Switcher (#6161)
Kong/docs.konghq.com: app/konnect/org-management/social-identity-login.md

---
title: Social Identity Login
content_type: how-to
---

Users have the option to register for and sign in to {{site.konnect_short_name}} accounts using social identity from the following providers:
- Google
- GitHub 
- Microsoft

## Account Linking

> It is possible to login with one or more social identity such as Google and Github. Upon login with subsequent identity, users are asked to link accounts such that multiple social identities are associated to the same {{site.konnect_short_name}} accounts. If the user does not choose to “Link Accounts”, each account will behave as its own independent identity and be associated with its own organizations. Organizations will only be associated with one social identity. It is not currently possible to re-link accounts. {{site.konnect_short_name}} recommends “Linking Accounts” when using multiple social identities.

Reviewer: ```suggestion
You can login with one or more social identity such as Google and Github. When you login with a different identity, you're asked to link accounts so your social identities are associated to the same {{site.konnect_short_name}} accounts. 

We recommend linking accounts when using multiple social identities, If you choose not to link, each account will behave as its own independent identity and be associated with its own organizations. Organizations will only be associated with one social identity. It is not currently possible to relink accounts. 
```

> ## Disable Social Login (Enterprise Only)

Reviewer: ```suggestion
## Disable Social Login
{:.badge .enterprise}
```
